### PR TITLE
chore(STONEO11Y-75): Add limitations to README

### PR DESCRIPTION
**Pull request by avi-biton**
- Create a README file
- Add limitations for `appstudio_container_network_transmit_bytes_total` and init_containers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
-Stonesoup observability 
+# RHTAP Observability 
+
+This repository contains Prometheus recording rule files and alert rule files for observability use, as well as unit tests for these rules and alerts.
+
+## Recording Rules
+
+- `appstudio_container_resource_limits` collects all the resources limits (cpu and memory) for all containers and init_containers, and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric
+
+- `appstudio_container_resource_minutes_gauge` uses the `appstudio_container_resource_limits` to create a 1 minutes gauge and calculate the limits according to the live time of the container
```
Comment (svatares) [inline README.md:9]: 1 minute (no 's').
Should "live time" be "lifetime" or "live-time" is a common expression?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+
+- `appstudio_container_network_transmit_bytes_total` collects the network egress of containers and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric.
+
+### Limitations
+
+`appstudio_container_network_transmit_bytes_total` is using cAdvisor's `container_network_transmit_bytes_total` to collect the data.  
+We encounetered several issues while testing it with pods containing init_containers that transmit data.
```
Comment (svatares) [inline README.md:16]: Did we resolve the issues we encountered? If so, should it be mentioned?

**Review by eisraeli — APPROVED** [review]
LGTM

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
-# o11y
-Stonesoup observability 
+# RHTAP Observability 
+
+This repository contains Prometheus recording rule files and alert rule files for observability use, as well as unit tests for these rules and alerts.
```
Comment (yftacherzog) [inline README.md:3]: alerts are rules as well
```suggestion
This repository contains Prometheus recording rule files and alert rule files for observability use, as well as unit tests for these rules.
```

File: README.md
Comment (yftacherzog) [inline README.md:1]: Can we break the lines at 89 characters or something similar?
Will make it easier to maintain.

File: README.md (hunk 1 above)
Comment (yftacherzog) [reply README.md:9]: `resources limits` -> `resource limits`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,21 @@
...
+
+## Recording Rules
+
+- `appstudio_container_resource_limits` collects all the resources limits (cpu and memory) for all containers and init_containers, and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric
```
Comment (yftacherzog) [inline README.md:7]:
```suggestion
- `appstudio_container_resource_limits` collects all the resource limits (cpu and memory) for all containers and init_containers, and adds the `label_pipelines_appstudio_openshift_io_type` label to the new metric.
```

File: README.md (hunk 1 above)
Comment (yftacherzog) [inline README.md:9]: `and calculate` -> `and calculates`

File: README.md (hunk 2 above)
Comment (yftacherzog) [reply README.md:16]: I think we should just mention the limitations rather than explaining the process we had.

File: README.md (hunk 1 above)
Comment (avi-biton) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (avi-biton) [reply README.md]: Done, I added the description of the limitation

File: README.md (hunk 3 above)
Comment (avi-biton) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -1,2 +1,21 @@
...
+
+- `appstudio_container_resource_minutes_gauge` uses the `appstudio_container_resource_limits` to create a 1 minutes gauge and calculate the limits according to the live time of the container
+
+- `appstudio_container_network_transmit_bytes_total` collects the network egress of containers and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric.
```
[the remark this answers is not in the document]
Comment (avi-biton) [reply README.md]: Done

File: README.md
Comment (avi-biton) [reply README.md:1]: Done

File: README.md [hunk 6]
```diff
@@ -1,2 +1,34 @@
-# o11y
-Stonesoup observability 
+# RHTAP Observability 
+
+This repository contains Prometheus recording rule files and alert rule files for  
+observability use, as well as unit tests for these rules.
```
Comment (yftacherzog) [inline README.md:4]:
```suggestion
This repository contains Prometheus recording rule files and alert rule files for
observability use, as well as unit tests for these rules.
```

Comment (avi-biton) [reply README.md:4]: Done, PTAL

**Review by yftacherzog — APPROVED** [review]
lgtm

**Pull request merged**